django2pydantic/handlers/json.py

```
@@ -1,15 +1,28 @@
 """Handler for JSON fields."""
 
-from typing import override
+import json
+from types import UnionType
+from typing import Annotated, Any, override
```

Pylint (3.3.7)

[error] No name 'override' in module 'typing'

(E0611)